packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeLink/urlConfig.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * This file contains the URLs for different platforms and environments.
+ * urlConfig will be sent as a proposal to the backend.
+ */
+
+/**
+ * URLs for the cTrader platform.
+ */
+export const ctraderLinks = {
+    android: 'https://play.google.com/store/apps/details?id=com.deriv.ct',
+    ios: 'https://apps.apple.com/us/app/deriv-ctrader/id6466996509',
+    live: 'https://ct.deriv.com',
+    mac: 'https://getctradermac.com/deriv/ctrader-deriv-setup.dmg',
+    staging: 'https://ct-uat.deriv.com',
+    windows: 'https://getctrader.com/deriv/ctrader-deriv-setup.exe',
+};
+
+/**
+ * URLs for the dxTrade platform.
+ */
+export const dxtradeLinks = {
+    android: 'https://play.google.com/store/apps/details?id=com.deriv.dx',
+    demo: 'https://dx-demo.deriv.com',
+    huawei: 'https://appgallery.huawei.com/app/C104633219',
+    ios: 'https://apps.apple.com/us/app/deriv-x/id1563337503',
+    live: 'https://dx.deriv.com',
+};
+
+/**
+ * URLs for the mt5 platform.
+ */
+export const whiteLabelLinks = {
```

Comment:
Don't we get all these from the API?

Reply:
When are we supposed to get the rest from the API? is there a card for it?

Reply:
the BE card is a WIP by Venugopal not sure about release